docs/content/Getting-Started/quickstart.md

```diff
@@ -345,7 +345,7 @@ Lastly, let's check that the workload is working in both clusters:
 For `florin`:
 
 ```shell
-curl http://localhost:8081
+curl http://localhost:8094
```

Comment:
Remember that the quickstart tells the user to use the bootstrap strip to download and unpack the most recent release, which is v0.2.2 at the moment, and use the kind Cluster objects in that release's `examples` directory.  So the changes in this PR to those `examples` files will not affect users yet.  So changing only the `curl` in the quickstart will create a broken quickstart.

I think the most likely fix is to expand this PR to change the quickstart so that it does not tell the user to use the kind Cluster objects in the `examples` directory but instead exhibits inline the `Cluster` YAMLs to use.